aws-datastore/src/main/java/com/amplifyframework/datastore/AWSDataStorePlugin.java

```diff
@@ -586,7 +588,7 @@ public void observe(
     @Override
     public <T extends Model> void observe(
             @NonNull Class<T> itemClass,
-            @NonNull String uniqueId,
+            @NonNull Serializable uniqueId,
```

Comment:
is this a breaking change for developers or do their existing `observe` calls continue to compile? ie. String is just a Serializable

Reply:
No since customer would be passing in the string which is an implementation of serializable.